### PR TITLE
Feature: Social Icons Integration on Login and Signup Page

### LINKED ISSUE #1304: Social Icons Integration on Login and Signup Page
**Is your feature request related to a problem? Please describe.**
yes, Improving social reach of the palisadoes organization

**Describe the solution you'd like**
Social icons of Talawa should be added to the login page and signup page of both user and admin portal, each redirecting to the respective social platform.

**Describe alternatives you've considered**
NA.

**Approach to be followed (optional)**
The login page should include social icons for all Talawa platforms, each redirecting to the respective platform.

**Additional context**
Screenshot 2023-12-28 at 12 25 20 AM

Add slack icon also

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by aialok**
**What kind of change does this PR introduce?**
- Added Social Icon of The Palisadoes Foundation for given platform.

**Issue Number:**

Fixes #1304 
Closes #1304 

**Did you add tests for your changes?**

- No

**Snapshots/Videos:**

[cinnamon-2023-12-31T004059+0530.webm](https://github.com/PalisadoesFoundation/talawa-admin/assets/66772290/ae722643-228e-49e0-924f-dc9e2ebed24c)

**If relevant, did you update the documentation?**

**Summary**

- Added Social Icon  on login and signup page of The Palisadoes Foundation

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by aashimawadhwa** [comment]
Please add the icons similar to the figma file @aialok

**Comment by aialok** [comment]
> Please add the icons similar to the figma file @aialok

Changes_admin

- Making changes that closely resemble the Figma file is a bit challenging. Are the above changes acceptable?

**Comment by aashimawadhwa** [comment]
Add the slack icon as well and please link the talawa slack to it @aialok

**Comment by aialok** [comment]
> Add the slack icon as well and please link the talawa slack to it @aialok 

Sure, should I also add Instagram ?

**Comment by rishav-jha-mech** [comment]
@aialok you need to add all the social media icons, and they should be colored as given in figma design

**Comment by aialok** [comment]
@aashimawadhwa , Please have a look.

Screenshot from 2024-01-01 01-13-51
Screenshot from 2024-01-01 01-13-58

File: src/screens/LoginPage/LoginPage.tsx [hunk 1]
```diff
@@ -223,6 +232,72 @@ function loginPage(): JSX.Element {
...
+                href="https://www.youtube.com/@PalisadoesOrganization"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <FaYoutube
+                  className={`${styles.socialIcon} ${styles.youtubeIcon}`}
+                />
+              </a>
+              <a
+                href="https://join.slack.com/t/thepalisadoes-dyb6419/shared_invite/zt-29oltereu-qJ931LcKxswuCAy29iA9WA"
```
Comment (palisadoes) [inline src/screens/LoginPage/LoginPage.tsx:283]: This URL is only valid for 30 days. It changes constantly. Use:
- https://www.palisadoes.org/slack

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comment

**Comment by rishav-jha-mech** [comment]
This is how the Login/Register screen should look like
Home screen _ LOGIN (1)

Also you need to add link to the Palisadoes Image (use anchor tag), clicking on which a user can go to the official website of Palisadoes.

The icons should be rounded, you can find the updated designs here on this figma link.

https://www.figma.com/file/dTGr5CnGHvkilU5nwnRXg2/Talawa-Admin-UI-Redesign?type=design&node-id=307-212&mode=design&t=qXBoOuK57MBGN4QZ-0

You can refer to this issue, for the Community Settings,
https://github.com/PalisadoesFoundation/talawa-admin/issues/1334

**Comment by aialok** [comment]
- I have a small suggestion. Since I am using SVGs directly from the Figma design, Can I create a folder named 'social-icons' under the SVGs parent directory and placing all SVGs inside it. This would provide ease of access and reduce potential conflicts for anyone working in the future."
- In the future, if there is a community that wants to add or delete their own social icons, they can easily do so by adding or deleting from the 'social-icons' folder.

@rishav-jha-mech

**Comment by aialok** [comment]
- Here is updated social-icon with figma design.

[cinnamon-2024-01-02T153259+0530.webm](https://github.com/PalisadoesFoundation/talawa-admin/assets/66772290/3b5b6bd1-3659-4e43-8bd5-c0e4630f0e98)

File: INSTALLATION.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ This document provides instructions on how to set up and start a running instanc
 - [Installation](#installation)
   - [Setting up this repository](#setting-up-this-repository)
   - [Setting up npm](#setting-up-npm)
+  - [Setting up Typescript](#setting-up-typescript)
```
Comment (rishav-jha-mech) [inline INSTALLATION.md:13]: No need to change this file

Comment (aialok) [reply INSTALLATION.md:13]: In the recent pull request, we added a new feature for an auto Table of Contents generator. There might be cases where some headings aren't included in the TOC, and as a result, these lines are part of this PR.

**Comment by aashimawadhwa** [comment]
this LGTM now.

**Review by aashimawadhwa — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**